crates/version/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "calimero-version"
+version = "0.1.0"
```

Comment:
now, this can relocate the versions specified in `merod` and `meroctl`, while we revert those to `.workspace = true`

Reply:
this is unresolved

Reply:
still pending